assets/js/components/TabbedNavigation.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = class TabbedNavigation {
+
+  constructor($elm, _window = window, doc = document) {
+    this.$elm = $elm;
+    this.window = _window;
+    this.doc = doc;
+    this.activeClassName = 'tabbed-navigation__tab-label--active';
+    this.$tabbedNavigationLinks = Array.prototype.slice.call(this.$elm.querySelectorAll('.tabbed-navigation__tab-label a'));
+    this.$tabbedNavigationLinks.forEach((tabbedNavigationLink) => {
+      tabbedNavigationLink.addEventListener('click', this.showActiveTab.bind(this));
+    });
+  }
+
+  showActiveTab(e) {
+    e.preventDefault();
+
+    const lastActiveElement = this.$elm.querySelector('.tabbed-navigation__tab-label--active a');
+
+    if (!lastActiveElement || e.target === lastActiveElement) {
+      return;
+    }
+
+    lastActiveElement.parentNode.classList.remove(this.activeClassName);
+    e.target.parentNode.classList.add(this.activeClassName);
+  }
```

Comment:
I'm not clear how this is supposed to work in production. Where is the javascript to perform the important operation of retrieving and displaying the content? What is the plan to get this ready for production?

Reply:
I think these tabs should do the same as View selector primary and secondary links - redirect to another page or "jump" to section with specific ID (for example click on a link with href=#articleUrl, will navigate to a section with the same ID).

Reply:
> I think these tabs should do the same as View selector primary and secondary links - redirect to another page or "jump" to section with specific ID (for example click on a link with href=#articleUrl, will navigate to a section with the same ID).

@chris-huggins would this be acceptable?

Reply:
@nlisgo Yes I think that makes sense. 

i.e. Figures and data should go to the separate page in the same way as the current selector 

We would need a new separate page for "Peer Review" similar to on EPP. We would need to move the content from the lower Peer Review and Author Response article sections to this new tab/page.

A final tab would switch to side-by-side view, which I guess is also a new page (but wouldn't have tabs introduced). 


I would like to avoid a tab causing scrolling down the page.